deploy/infra/lib/constructs/app-backend.ts

```diff
@@ -167,6 +166,18 @@ export class AppBackend extends Construct {
       })
     );
 
+    this._lambda.addToRolePolicy(
```

Comment:
This policy is up for debate, I have proposed using a tag condition to restrict this to only assume roles which have the specific tag

Not settled on what the tag key/value should be